stable/voyager/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Voyager provides controller for Ingress and Certificates for Kubernetes developed by AppsCode.
 icon: https://cdn.appscode.com/images/icon/voyager.png
 name: voyager
-version: 0.1.0
+version: 0.2.0
```

Comment:
If they were always them same, we didn't need `appVersion`. You may always update a chart without updating the image version. You may have a hard time keeping those in sync. People may always create PRs to add some functionality or fix things. You cannot really control this.